web/v1/snowflake/sql-runner/sql/standard/00-setup/01-main/01-stored-procedures.sql

```diff
@@ -156,139 +160,111 @@ CREATE OR REPLACE PROCEDURE {{.output_schema}}.column_check(SRC_SCHEMA  VARCHAR,
-  DRV_JOIN_KEY:              the join key for derived table
-  DRV_PARTITION_KEY:         the partition key for derived table
-  DRV_UPSERT_LOOKBACK FLOAT: corresponds to {{.upsert_lookback_days}}::float
-
+  STAGE_NEXT:            corresponds to '{{.stage_next}}'
+  AUTOMIGRATE:           whether to automigrate staged table
+  STAGING_SOURCE:        the staging source table
+  STAGING_TARGET:        the staging target table
+  STAGING_JOIN_KEY:      the join key for staged tables
+  SKIP_DERIVED:          corresponds to '{{.skip_derived}}'
+  DERIVED_SOURCE:        the derived source table
+  DERIVED_TARGET:        the derived target table
+  DERIVED_JOIN_KEY:      the join key for derived table
+  DERIVED_PARTITION_KEY: the partition key for derived table
+  UPSERT_LOOKBACK FLOAT: corresponds to {{.upsert_lookback_days}}::float
   To drop:
-  DROP PROCEDURE {{.output_schema}}.commit_with_metadata(VARCHAR,VARCHAR,VARCHAR,VARCHAR,
-                                                         VARCHAR,VARCHAR,VARCHAR,VARCHAR,
-                                                         VARCHAR,VARCHAR,VARCHAR,VARCHAR,
-                                                         VARCHAR,VARCHAR,VARCHAR,FLOAT);
-
+  DROP PROCEDURE {{.output_schema}}.commit_tables(VARCHAR,VARCHAR,VARCHAR,VARCHAR,
+                                                  VARCHAR,VARCHAR,VARCHAR,VARCHAR,
+                                                  VARCHAR,VARCHAR,FLOAT);
 */
-CREATE OR REPLACE PROCEDURE {{.output_schema}}.commit_with_metadata(
-                                                        META_TBL            VARCHAR,
-                                                        META_FROM_TBL       VARCHAR,
-                                                        STAGE_NXT           VARCHAR,
-                                                        STG_SRC_SCHEMA      VARCHAR,
-                                                        STG_SRC_TBL         VARCHAR,
-                                                        STG_TRG_SCHEMA      VARCHAR,
-                                                        STG_TRG_TBL         VARCHAR,
-                                                        STG_JOIN_KEY        VARCHAR,
-                                                        SKIP_DRV            VARCHAR,
-                                                        DRV_SRC_SCHEMA      VARCHAR,
-                                                        DRV_SRC_TBL         VARCHAR,
-                                                        DRV_TRG_SCHEMA      VARCHAR,
-                                                        DRV_TRG_TBL         VARCHAR,
-                                                        DRV_JOIN_KEY        VARCHAR,
-                                                        DRV_PARTITION_KEY   VARCHAR,
-                                                        DRV_UPSERT_LOOKBACK FLOAT)
+CREATE OR REPLACE PROCEDURE {{.output_schema}}.commit_tables(
```

Comment:
As we discussed, doing both in one might have a negative impact on customer experience - because it's less intuitive. 

However Snowflake say that it's more performant to use one transaction instead of two if possible.

So, we'll test if there's a performance impcat to separating the two, and if not we can break this down to be more atomic and simpler. If so, it can stay as is.